lib/screens/tutorial.dart

```diff
@@ -1,32 +1,137 @@
 
   final String title;
+  //Each prompt represents a new animated text bubble
+  final String firstPrompt = tutorialText["firstScreen"]!;
+  final String secondPrompt = tutorialText["secondScreen"]!;
+  final String thirdPrompt = tutorialText["thirdScreen"]!;
   static const routeName = "Tutorial";
 
   @override
   State<Tutorial> createState() => _TutorialState();
 }
 
 class _TutorialState extends State<Tutorial> {
+  //Tracks which screen to utilize
+  int screenCounter = 0;
+  //Saves string to be used for animated text
+  String currentString = "";
+  //Saves path to image asset for displaying map
+  String asset = "";
+
+  @override
+  void initState() {
+    super.initState();
+    currentString = widget.firstPrompt;
+  }
+
   @override
   Widget build(BuildContext context) {
-    User? user = FirebaseAuth.instance.currentUser;
     return Scaffold(
-      body: Container(
-          constraints: const BoxConstraints.expand(),
-          decoration: const BoxDecoration(
-            image: DecorationImage(
-                image:
-                    AssetImage("assets/images/logos_and_icons/background.png"),
-                fit: BoxFit.cover),
-          ),
+        body: Container(
+      constraints: const BoxConstraints.expand(),
+      decoration: const BoxDecoration(
+        image: DecorationImage(
+            image: AssetImage("assets/images/logos_and_icons/background.png"),
+            fit: BoxFit.cover),
+      ),
+      child: Container(
+          padding: EdgeInsets.only(
+              top: MediaQuery.of(context).size.height * 0.25,
+              bottom: MediaQuery.of(context).size.height * 0.25),
           child: Column(
-              mainAxisAlignment: MainAxisAlignment.center,
-              children: [Text(user!.displayName!)])),
-    );
+            children: [
+              spaceHolder(context, asset),
+              const Padding(padding: EdgeInsets.only(bottom: 30)),
+              Row(
+                children: [
+                  Expanded(
+                    child: Column(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      crossAxisAlignment: CrossAxisAlignment.center,
+                      children: [
+                        Image.asset("assets/images/logos_and_icons/leaf1.png")
+                      ],
+                    ),
+                  ),
+                  Expanded(
+                    child: GestureDetector(
+                      //String used in customText is set here.
+                      //Each tap increments the counter, and ultimately routes to
+                      //game map.
+                      onTap: () {
+                        screenCounter++;
+                        if (screenCounter == 3) {
+                          Navigator.of(context)
+                              .pushNamed(MapViewScreen.routeName);
+                        }
+                        setState(() {
+                          if (screenCounter == 1) {
```

Comment:
If you use a different data structure for this, you can avoid this if/else business:
eg.
```

var tutorials = [
  {
    "prompt":
        '''Welcome to the world of Math Go, bla. This is a brief tutorial to help you get acclimated to the world around you. Click on this chat bubble when you are ready to proceed.''',
    "asset": ""
  },
  {
    "prompt":
        '''The game map will display your current position and any beasties nearby. If you tap a beastie, you will be prompted for a question. Answer it correctly, and the beastie is yours!. Be warned, if you input the wrong answer, the beastie will flee!''',
    "asset": "assets/images/logos_and_icons/sample_map.png"
  },
  {
    "prompt": '''That's all there is to it! Good luck and have fun!''',
    "asset": "assets/images/logos_and_icons/question.png"
  }
];
```
Then you can do something like
```
                        setState(() {
                            currentString = tutorials[screenCounter]['prompt'];
                            asset = tutorials[screenCounter]['asset'];
                        });
                        ```

Reply:
Changed code to model example.